pkgs/ffigen/CHANGELOG.md

```diff
@@ -6,6 +6,20 @@
 - Add `retainAndReturnPointer` method to ObjC objects and blocks, and add
   `castFromPointer` method to blocks.
 - Add `-Wno-nullability-completeness` as default compiler option for MacOS.
+- __Breaking change__: Use `package:objective_c` in ObjC bindings.
+  - ObjC packages will have a flutter dependency (until #1068 is fixed).
```

Comment:
Nit #1068 prefer using a full URL, then things keep working if we ever migrate repos again.

Reply:
Done